src/__tests__/fireEvent.test.tsx

```diff
@@ -11,33 +10,49 @@ import {
 } from 'react-native';
 import { render, fireEvent } from '..';
 
-const OnPressComponent = ({ onPress, text }) => (
+const OnPressComponent = ({
```

Comment:
Consider extracting `XxxProps` types for all of these components, pls place them next to each component.